CHANGELOG.md

```diff
@@ -9,6 +9,9 @@ See [Conventional Commits](https://conventionalcommits.org) for commit guideline
 ### Do not collect XHR events for data URLs
 AJAX events for data URLs have not historically been collected due to errors in the agent when handling URLs without hostnames. Going forward, XHR calls to data URLs will not cause agent errors and will continue to be excluded from collection.
 
+### Reduce size of builds for modern browser targets
+The agent will now transpile outputs for browsers with ES6 support separately from IE11, reducing loader size for these browsers by 20% or more. For the standard build, babel will now target the last ten versions of Chrome, Edge, Safari, and Firefox. See [browser agent EOL policy](https://docs.newrelic.com/docs/browser/browser-monitoring/getting-started/browser-agent-eol-policy/).
```

Comment:
I'd like to suggest a rewording for concise:
```suggestion
The agent is now compatible with _only modern web syntax (ES6+)_; **this reduces loader size for these browsers by 20% or more**. Our standard build targets and tests support for just the last ten versions of Chrome, Edge, Safari, and Firefox -- see [browser agent EOL policy](https://docs.newrelic.com/docs/browser/browser-monitoring/getting-started/browser-agent-eol-policy/) for more details.
```

Reply:
For majority customer facing perspective, there's only "one build" of the agent, which is the standard build. We don't support or give the polyfill+es5 to any other entity aside from [redacted].